.travis.yml

```diff
@@ -150,6 +150,12 @@ matrix:
           compiler: gcc
         - arch: arm64
           os: osx
+        # arm64 times out when using -fsanitize=address
+        # The amd64 build should be sufficient to test this.
+        # Note the env line must exactly match the env setting above 
```

Comment:
It took me a while to understand the meaning of line 155. How about making the following clarification?

```
# Note the env line must exactly match the env line from the build
# matrix above which contains the `-fsanitize=address` option.
```

Reply:
Hmm, this gives me the heeby jeebies, someone is bound to make a mistake some day (say, update the config higher up without modifying it accordingly here). I guess it can be OK now, but should be refactored later on. Please add a TODO comment to that effect.

Reply:
Updated the comment : Also added another note above the actual build matrix entry so that it might be read :)